src/palace/manager/feed/acquisition.py

```diff
@@ -212,7 +212,7 @@ def _create_entry(
         cls,
         work: Work,
         active_licensepool: LicensePool | None,
-        edition: Edition,
+        edition: Edition | None,
```

Comment:
Because presentation_edition can be `None`, this type hint needed to be updated.